src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,88 @@
+package duke.task;
+
+import duke.parser.Parser;
+
+import java.util.ArrayList;
+
+public class TaskManager {
```

Comment:
As the TaskManager is performing a number of important functions, it would help to provide more [method headers](https://se-education.org/guides/conventions/java/basic.html#comments) here to describe what each one if doing.